### DISCUSSION TITLE
Add Status Page as Dashboard that all users including Basic can view.

### DISCUSSION THREAD

**Discussion by RFXTECH** [newrelic/nr1-status-pages#160] [Q&A] [answered]
Thanks for the awesome work on this project!   I was able to install it in our NewRelic account and get it configured will all the vendor status pages I care about.   It looks great being able to see at a quick glance status of all vendor services!   But when I tried to display it our monitoring TVs the dashboard account couldn't access it, also no one on our helpdesk could view the page either.   Is it possible to add this as a dashboard where all basic users can display it?

**Comment by aso1124** [comment, marked as answer]
Hi @RFXTECH - glad you were able to get the app up and running! Unfortunately, to include this view in a dashboard, we'd have to leverage a custom widget, and it would be subject to the same restrictions as the app - apps and custom widgets require at least a core user license level to access.

**Comment by khpeet** [comment]
https://github.com/newrelic/nr-labs-widget-pack#status-page - Core users and above can access
